tests/external-links.js

```diff
@@ -364,28 +364,32 @@ async function updateGithubIssue(urlResults) {
    */
   function getBodyFromLinkData(linkData) {
     const scriptName = import.meta.url.split(`/`).slice(-2).join(`/`);
+    const rows = Object.entries(linkData).map(([url, statuses]) => {
+      const statusIcons = statuses.map(status => {
+        if (!status.failed) {
+          return `:heavy_check_mark:`;
+        }
+
+        const message = status.message.replaceAll(`\n`, ` `).replaceAll(`"`, `&quot;`);
+        const emoji = getFailedEmoji(status.message);
+        return `<a href="${status.jobUrl}" title="${message}">${emoji}</a>`;
+      }).join(`&nbsp;`);
+      const link = `<a href="${url}" target="_blank">${url}</a>`;
+      return `<tr><td>${link}</td><td nowrap>${statusIcons}</td></tr>`;
```

Comment:
I thought you wanted to swap the columns? :smile: 

```suggestion
      return `<tr><td nowrap>${statusIcons}</td><td>${link}</td></tr>`;
```

Reply:
What a spanner 🤣

Reply:
Oh, what does that phrase mean?

Reply:
Sorry, it's a northern british phrase. "What a tool" or "What a spanner" or similar just means I realise what a silly mistake I've made 😂